src/__mocks__/addTask.js

```diff
@@ -0,0 +1,19 @@
+// export const addTask = () => {
+//   class Task {
+//     constructor(description) {
+//       this.description = description;
+//       this.index = new Date();
+//       this.completed = false;
+//     }
+//   }
+//   const addTaskForm = document.querySelector('#add-task');
+//   const data = storage
+//   addTaskForm.addEventListener('click', () => {
+//     const inputTaskValue = document.querySelector('#add-task-input').value;
+//     if (inputTaskValue === '') return;
+//     const newTask = new Task(inputTaskValue);
+//     data.push(newTask);
+//     localStorage.setItem('Task-list', JSON.stringify(data));
+//   });
+//   return data;
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments.